### PR TITLE
fix(shadow-database): Streamline document

### PR DESCRIPTION
**Pull request by janpio**
Streamline the content of the shadow database documentation.

### CODE REVIEW

**Comment by janpio** [comment]
Another approach, can probably be combined: https://github.com/prisma/docs/pull/4407

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 1]
```diff
@@ -46,44 +41,34 @@ When you run `prisma migrate dev` to create a new migration, Prisma Migrate uses
...
 
-Assuming Prisma Migrate did not [detect schema drift](#detecting-schema-drift), it moves on to generating new migrations from schema changes. To generate new migrations, Prisma Migrate:
+Assuming Prisma Migrate did not [detect schema drift](#detecting-schema-drift), it moves on to generating new migrations from Prisma schema changes. To generate new migrations, Prisma Migrate:
 
 1. Calculates the target database schema as a function of the current Prisma schema.
 1. Compares the end state of the existing migration history and the target schema, and generates steps to get from one to the other.
 1. Renders these steps to a SQL string and saves it in the new migration file.
-1. Applies the generated migration to the development database (assuming you have not specified the `--create-only` flag)
-1. Drops the shadow database (cloud-hosted databases cannot be dropped, but are reset at the start of the `migrate dev` command)
+2. TODO evaluate data loss
```
Comment (janpio) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]: TODO

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 2]
```diff
@@ -12,22 +12,22 @@ The shadow database is a second, _temporary_ database that is **created and dele
 
 <FootNote>
 
-\* If you use a cloud-hosted database for development, you need to [create the shadow database manually](#cloud-hosted-shadow-databases-must-be-created-manually).
+\* If your database does not allow creation and deleting of databases (e.g. in a cloud-hosted environment), you need to [create and configured the shadow database manually](#cloud-hosted-shadow-databases-must-be-created-manually).
```
Comment (janpio) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]:
```suggestion
\* If your database does not allow creation and deleting of databases (e.g. in a cloud-hosted environment), you need to [create and configure the shadow database manually](#cloud-hosted-shadow-databases-must-be-created-manually).
```

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 3]
```diff
@@ -41,42 +41,32 @@ When you run `prisma migrate dev` to create a new migration, Prisma Migrate uses
...
 
 ### Generating new migrations
 
-Assuming Prisma Migrate did not [detect schema drift](#detecting-schema-drift), it moves on to generating new migrations from schema changes. To generate new migrations, Prisma Migrate:
+Assuming Prisma Migrate did not [detect schema drift](#detecting-schema-drift), it moves on to generating new migrations from Prisma schema changes. To generate new migrations, Prisma Migrate:
 
 1. Calculates the target database schema as a function of the current Prisma schema.
 1. Compares the end state of the existing migration history and the target schema, and generates steps to get from one to the other.
 1. Renders these steps to a SQL string and saves it in the new migration file.
+1. Evaluate data loss caused by the SQL and warns about that
```
Comment (nikolasburk) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]: just making punctuation consistent with other bullets:

```suggestion
1. Evaluate data loss caused by the SQL and warns about that.
```

**Review by nikolasburk — APPROVED** [review]
looks great, thanks Jan! 🎉

**Pull request merged**